application-forum-ui/src/main/resources/ForumCode/NewFlagSheet.xml

```diff
@@ -60,26 +60,46 @@
   ## sending email
   #set($emailRegex = "[-0-9a-zA-Z.+_]+@[-0-9a-zA-Z.+_]+\.[a-zA-Z]{2,4}")
   #set($adminEmail = $xwiki.getXWikiPreference('admin_email'))
+  #if ("${adminEmail}" == '')
+    ## Get admin email from global administration settings
+    #set ($configDocName = 'Mail.MailConfig')
+    #set ($configDoc = $xwiki.getDocument($configDocName))
+    #set ($configClassName = 'Mail.SendMailConfigClass')
+    #set ($configObj = $configDoc.getObject($configClassName))
+    #set ($adminEmail = $configObj.from)
+  #end
   #if (!$adminEmail.matches($emailRegex))
     #set($adminEmail = '')
   #end
-  #set($moderator = $flagDoc.getValue('moderator'))
-  #if("$!{moderator}" != '')
-    #set($moderatorEmail = $xwiki.getUserName($moderator, "$email", false))
-    #if(!$moderatorEmail.matches($emailRegex))
-      #set($moderatorEmail = '')
+
+  #set ($moderator = $flagDoc.getValue('moderator'))
+  #if ("$!{moderator}" != '')
+    #set ($moderatorEmail = $xwiki.getUserName($moderator, "$email", false))
+    #if (!$moderatorEmail.matches($emailRegex))
+      #set ($moderatorEmail = '')
     #end
   #end
 
   == $services.localization.render('conversations.flag.success.dialog.header') ==
 
   $services.localization.render('conversations.flag.success.dialog.message')
 
-  #if("$!{adminEmail}" != '' &amp;&amp; "$!{moderatorEmail}" != '')
-    #set($mailTo = "${adminEmail},${moderatorEmail}")
-    #set($url = $flagDoc.getExternalURL())
-    #set($res = $xwiki.mailsender.sendMessageFromTemplate($adminEmail, $mailTo, $util.null, $util.null, '', 'ForumCode.MailTemplateFlag', $xcontext.vcontext))
-    #if ($res != 0)
+  #if ("$!{adminEmail}" != '')
+    #set ($mailTo = "${adminEmail}")
```

Comment:
You can always use an array if the string you're concatenating would behave like an array :)

Reply:
`$mailTo` is not actually an array, it's used like an email list ( e.g `email1, email2`). It would be another possibility to use an array & join it using `stringtool.join`. Considering that, the worst case is an array with 2 elements, so I'm not sure if the approach using array would be a better option (excluding the readability factor).